scripts/e2e/bootstrap_job.template

```diff
@@ -9,6 +9,13 @@ spec:
       tolerations:
       - effect: NoSchedule
         key: node-role.kubernetes.io/master
+      initContainers:
+      - name: clusterctl-bin
+      image: gcr.io/cnx-cluster-api/clusterctl:${VERSION}
+      command: ['sh', '-c', 'cp /root/clusterctl /tmp/clusterctl/clusterctl && chmod +x /tmp/clusterctl/clusterctl']
```

Comment:
If you're going to do this, is it possible to copy it to tmp/clusterctl/clusterctl-${VERSION} just to be clear what version we're putting into tmp.